kratos.gid/apps/EmbeddedFluid/python/KratosFluid.py

```diff
@@ -76,11 +80,12 @@
     FluidModel.AddModelPart(main_model_part.GetSubModelPart(gravity_part_name))
 
 ## Print model_part and properties
-if(verbosity > 1):
-    print("")
-    print(main_model_part)
-    for properties in main_model_part.Properties:
-        print(properties)
+if(echo_level > 1):
+    if (parallel_type == "OpenMP") or (mpi.rank == 0):
+        print("")
+        print(main_model_part)
+        for properties in main_model_part.Properties:
```

Comment:
Probably we can remove this one, or maybe print it when echo_level is larger than one (as is done with other print statements).